src/components/tools/geometry-tool/geometry-content.tsx

```diff
@@ -941,7 +941,7 @@ export class GeometryContentComponent extends BaseComponent<IProps, IState> {
           if (parsedChange) {
             const result = content.applyChange(board, parsedChange);
             if (result) {
-              this.handleCreateElement(result as JXG.GeometryElement);
+              this.handleCreateElements(result as JXG.GeometryElement | JXG.GeometryElement[]);
```

Comment:
How did this work before, line copy-paste isn't the first change to return multiple objects, right?

Reply:
I'm not sure. I was surprised that this hadn't already been done.